components/copy-text-dialog.vue

```diff
@@ -3,13 +3,21 @@
     v-model="dialogOpen"
     max-width="400">
     <v-card>
-      <v-card-title
-        class="headline"
-        primary-title>
-        Kopieren
-      </v-card-title>
 
-      <v-card-text>
+      <v-toolbar
+        dark
+        color="primary">
+        <v-toolbar-title>Teilen</v-toolbar-title>
+        <v-spacer />
+        <v-btn
+          icon
+          dark
+          @click.native="dialogOpen = false">
+          <v-icon>close</v-icon>
```

Comment:
Im Bearbeiten-Dialog ist das X links, in diesem rechts

Reply:
Sah nicht so hübsch aus weil "Teilen" so kurz war als wort, jetzt ist er links und der Titel heißt "Link Teilen". So ist es annehmbar.